DESCRIPTION

```diff
@@ -0,0 +1,23 @@
+Package: IOHexperimenter
+Type: Package
+Title: Benchmarking Part of the 'IOHprofiler'
+Version: 0.0.0.9000
+Author: Furong Ye [cre, aut], Hao Wang [aut], Diederick Vermetten [aut], Carola Doerr [aut], Thomas Bäck [aut]
```

Comment:
I would suggest putting you as the First author / creator

Reply:
The official meaning of the [cre] tag seems to be:
the creator or maintainer, the person you should bother if you have problems.

It seems to be possible to put multiple creators, so should I just put you and Furong as creators as well?

Reply:
Okay, I suggest the following:  Diederick Vermetten [cre, aut], Furong Ye [cre, aut], Hao Wang [aut], Carola Doerr [aut], Thomas Bäck [aut]